### PR TITLE
Problem: no integration test on filter apis (fix #283)

### LINKED ISSUE #283: Problem: no integration test on filter apis
- [x] [pending filters](https://github.com/crypto-org-chain/cronos/issues/403)
- [ ] logs filter
- [ ] block filter
- etc.

### PR DESCRIPTION
**Pull request by linfeng-crypto**

### CODE REVIEW

File: integration_tests/test_filters.py [hunk 1]
```diff
@@ -12,4 +19,34 @@ def test_pending_transaction_filter(cluster):
...
+    assert "Hello" == mycontract.caller.greet()
+    current_height = hex(w3.eth.get_block_number())
+    event_filter = mycontract.events.ChangeGreeting.createFilter(fromBlock=current_height)
+
+    tx = mycontract.functions.setGreeting("world").buildTransaction()
+    tx_receipt = send_transaction(w3, tx)
+    log = mycontract.events.ChangeGreeting().processReceipt(tx_receipt)[0]
+    assert log["event"] == "ChangeGreeting"
+    assert tx_receipt.status == 1
+    new_entries = event_filter.get_new_entries()
```
Comment (thomas-nguy) [inline integration_tests/test_filters.py:49]: can also check the length here

**Review by thomas-nguy — APPROVED** [review]
lgtm otherwise

File: integration_tests/test_filters.py (hunk 1 above)
Comment (linfeng-crypto) [reply integration_tests/test_filters.py:50]: Ok

**Review by yihuang — APPROVED** [review]
LGTM

**Pull request merged**